docs/docs/methods.md

```diff
@@ -1614,3 +1614,63 @@ gte(1, 2)  // false
 gte(2, 2)  // true
 gte(3, 2)  // true
 ```
+
+## `has`
+
+Checks if an object has provided own property
+
+```js
+import { has } from 'nanoutils'
+
+const obj = {
+  a: 1,
+  __proto__: {
+    b: 2
+  }
+}
+
+has('a', obj)   // true
+has('b', obj)   // false
+```
+
+::: tip
+By own it means that `prototype` is not used to find a property
```

Comment:
I suggest another tip, because current one has some mistakes:
"Method does not use prototype chain, only object itself".